### DISCUSSION TITLE
en/bing-jail/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#105] [General]
# en/bing-jail/

A short story on how my website lost all indexed pages in Bing, affecting my presence in DuckDuckGo, secret blacklisting and classification as a splog by Bing AI

https://dariusz.wieckiewicz.org/en/bing-jail/

**Comment by wrq** [comment]
So, based on your choosing not to follow the (freely available and accessible) Bing guidelines, why should they bother with support for a non-issue like this? Is there some reason you are entitled to free indexation by a search engine? Do you pay Bing for this service?

I just find this whole thing sort of ridiculous because you've outlined a half dozen things Bing wants fixed to index you, and you dismiss half of them with no explanation at all, and then Bing is supposed to be the bad guy? Your site looks like spam to an automated system and despite a literal recipe of how to fix it, you refuse. The end result? Your site is treated like spam. 

It's actually insane to me that you note yourself as knowledgeable about SEO, because here the SE is giving you step-by-step instructions on how to O, and you've chosen to blog about it instead. 

If I were you, I'd delete this post, make the changes, and go back to doing what was working for you before.

**Reply by idarek** [reply]
The thing is that I am following Bing's guidelines. From 2.6k pages with 277 with warnings that don't justify the reason for non-indexation. The fact is that somebody is doing some shady business here delisting blogs and classifying them as spam. There are other people with more popular blogs that faced that same. 

I would refer you to Dave [Bing Webmaster Guidelines Compliance Report](https://daverupert.com/2023/01/my-bing-webmaster-guidelines-compliance-report/). Mine would look exactly the same so why the discrimination?

It's not possible to fix every single issue reported. Having 2,600 to 277 gives us how much? In my opinion not enough to discourage the whole website. And what is the point of Webmaster Tools that will not give you accurate feedback on why you are blacklisted?

Bing follows guidelines (like too long titles) that are classed outdated by many, but still, that's, not the reason to wipe from existence. Overall they are WARNINGS, not errors.

If you are so eager to call my site spam, please evaluate how?

**Comment by Lucire** [comment]
Hi Darius, I have been tracking this for about a year when one of our sites plunged from thousands to 50 entries on DDG. Like you, we used Duck Duck Go for our internal site search. I began noticing something was up when the searches were coming up empty. We switched to Mojeek, which IMO is better than Google when it comes to internal site searches, but I’ll come back to that.

I went through everything you did and *after* signing up to Bing Webmaster Tools, things got a lot worse. Bing ignored the site maps, and IndexNow (which we have access to) was a joke. Bing was basically finding and indexing things of great antiquity, This site is over 25 years old and Bing was showing pages from 1999 to 2005 for the most part. Nothing after 2009 except on rare occasions, then the entry would vanish again. (This particular aspect has improved since, but this was the case for most of 2022.)

Very, very few people were talking about it. Many kept thinking it had something to do with their submission process. The Microsoft forums are filled with puzzled webmasters and copy-and-paste respondents blaming them. But the reality is probably much simpler: Bing’s index has collapsed, and no one wants to talk about it.

I suspect this through doing a bunch of site: searches which I have entered on my blog. I don’t want to spam your comment form but I will be happy to share my links on request. (I don’t use my real name as my username on Git but you might be able to figure it out.) When Mojeek and Google have millions of pages for site:microsoft.com and Bing manages hundreds of thousands, then you know something is wrong. (A site:nytimes.com search nets only thousands of results on Bing, not that it’s capable of displaying anywhere near that many.) Also if you check out a certain Dutch academic’s site, where he does test queries and sees how many Google and Bing return, there are periods during 2022 where Bing flatlines. Again, happy to share that link if you want to have a look. When there are results, Bing often repeats them from page to page to pad out the index. Looking at the top 50 of some sites where I’ve done a site: search, I estimate 40 per cent of the results are repeats. That’s how tiny Bing’s index is now.

Have you heard of Techdirt? At one point, site:techdirt.com returned *one* result. Here is a well respected online publication that has run for over a quarter-century, and Bing (and its clones, Duck Duck Go, Qwant, Neeva, Yahoo, etc.) managed only one. I did blog about it and alerted the site owner and all of a sudden it began rising again. But if it can only manage one result for Techdirt, then you wonder how the rest of us might fare.

Google, meanwhile, began following Bing into antiquity and it now has huge trouble ranking dynamic pages over static ones. It shouldn’t make a difference but the research tells another story. In the top 10 for our site: searches, Google began bringing up pages from the 2000s and the number of PHP pages versus HTML ones in the top 50 went from nine in September to one last week. With Mojeek, it just behaves as a normal search engine should.

I hope this at least shows you are not alone and we can definitely level the blame at Microsoft and Bing.
